src/sfdt/__tests__/toggleBookmark.test.js

```diff
@@ -129,23 +129,19 @@ describe('toggleBookmark', function() {
 
 		test('Toggle off', function() {
 			let newSfdt = JSON.parse(JSON.stringify(bookmarkStartEndingInDifferentInline));
-			const toggleOn = toggleBookmark(
-				newSfdt,
-				'COND::9e7d0dc1-b9ed-4baa-9399-a4c4c9be96d4',
-				true
-			);
-			const firstInlineAfterToggle = getInline(toggleOn, 0);
-			const lastInlineAfterToggle = getInline(toggleOn, 0, 2);
+			// const toggleOn = toggleBookmark(
```

Comment:
you need to put this back no ?

Reply:
This is unused part in the test. I think the last changes was either by you or HC
So, removed this for lint issues, as they are not used in tests
Can you check once, why it was put there? If not required anymore, I'll remove those